app/services/mongodb/MongoDBFileService.scala

```diff
@@ -909,21 +909,21 @@ class MongoDBFileService @Inject() (
   }
 
   def searchAllMetadataFormulateQuery(requestedMetadataQuery: Any): List[File] = {
-    Logger.debug("top: "+ requestedMetadataQuery.asInstanceOf[java.util.LinkedHashMap[String,Any]].toString()  )
+    Logger.debug("top: " + requestedMetadataQuery.asInstanceOf[java.util.LinkedHashMap[String,Any]].toString()  )
     var theQuery =  searchMetadataFormulateQuery(requestedMetadataQuery.asInstanceOf[java.util.LinkedHashMap[String,Any]], "all")
-    Logger.debug("thequery: "+theQuery.toString)
+    Logger.debug("thequery: " + theQuery.toString)
     FileDAO.find(theQuery).toList
   }
 
   def searchUserMetadataFormulateQuery(requestedMetadataQuery: Any): List[File] = {
-    Logger.debug("top: "+ requestedMetadataQuery.asInstanceOf[java.util.LinkedHashMap[String,Any]].toString()  )
+    Logger.debug("top: " + requestedMetadataQuery.asInstanceOf[java.util.LinkedHashMap[String,Any]].toString()  )
     var theQuery =  searchMetadataFormulateQuery(requestedMetadataQuery.asInstanceOf[java.util.LinkedHashMap[String,Any]], "userMetadata")
-    Logger.debug("thequery: "+theQuery.toString)
+    Logger.debug("thequery: " + ßtheQuery.toString)
```

Comment:
Same here, looks liked there is an extra ß

Reply:
This is corrected, sorry about that. Probably was an accidental keystroke.